src/latticpp/ckks/ciphertext.cpp

```diff
@@ -13,6 +13,10 @@ namespace latticpp {
         return lattigo_ciphertextScale(ct.getRawHandle());
     }
 
+    int degree(const Ciphertext &ct) {
```

Comment:
Make this return a uin64_t, or add an explicit cast here from the `uint64_t` returned from Go (see comment above). I want it to be very obvious that the integer types align across language boundaries.